docs/guides/jupyter.rst

```diff
@@ -0,0 +1,398 @@
+
+Finally, you can include the notebook in any *toctree*.
+For example, add this to your root document:
+
+.. code-block:: rest
+
+   .. toctree::
+      :maxdepth: 2
+      :caption: Contents:
+
+      notebooks/Example 1
+
+The notebook will render as any other HTML page in your documentation
+after doing ``make html``.
+
+.. figure:: /_static/images/guides/example-notebook-rendered.png
+   :width: 80%
+   :align: center
+   :alt: Example Jupyter notebook rendered on HTML by nbsphinx
+
+   Example Jupyter notebook rendered on HTML by nbsphinx
+
+To further customize the rendering process among other things,
+refer to the nbsphinx_ or MyST-NB_ documentation.
+
+Rendering interactive widgets
+-----------------------------
+
+Widgets are eventful python objects that have a representation in the browser
+and that you can use to build interactive GUIs for your notebooks.
+Basic widgets using `ipywidgets`_ include controls like sliders, textboxes, and buttons,
+and more complex widgets include interactive maps, like the ones provided by `ipyleaflet`_.
+
+.. _ipywidgets: https://ipywidgets.readthedocs.io/
+.. _ipyleaflet: https://ipyleaflet.readthedocs.io/
+
+You can embed these interactive widgets on HTML Sphinx documentation.
+For this to work, it's necessary to *save the widget state*
+before generating the HTML documentation,
+otherwise the widget will appear as empty.
+Each editor has a different way of doing it:
+
+- The classical Jupyter Notebook interface
+  provides a "Save Notebook Widget State" action in the "Widgets" menu,
+  :ref:`as explained in the ipywidgets
+  documentation <ipywidgets:/embedding.md#embedding-widgets-in-html-web-pages>`.
+  You need to click it before exporting your notebook to HTML.
+- JupyterLab provides a "Save Widget State Automatically" option in the "Settings" menu.
+  You need to leave it checked so that widget state is automatically saved.
+- In Visual Studio Code `it's not possible to save the widget
+  state <https://github.com/microsoft/vscode-jupyter/issues/4404>`_
+  at the time of writing (June 2021).
+
+.. figure:: /_static/images/guides/jupyterlab-save-widget-state.png
+   :width: 30%
+   :align: center
+   :alt: JupyterLab option to save the interactive widget state automatically
+
+   JupyterLab option to save the interactive widget state automatically
+
+For example, if you create a notebook with a simple
+:ref:`ipywidgets:/examples/widget list.ipynb#intslider`
+widget from ipywidgets and save the widget state,
+the slider will render correctly in Sphinx.
+
+.. figure:: /_static/images/guides/widget-html.gif
+   :width: 80%
+   :align: center
+   :alt: Interactive widget rendered in HTML by Sphinx
+
+   Interactive widget rendered in HTML by Sphinx
+
+To see more elaborate examples:
+
+- `ipyleaflet`_ provides several widgets for interactive maps,
+  and renders live versions of them `in their
+  documentation <https://ipyleaflet.readthedocs.io/en/latest/api_reference/velocity.html>`_.
```

Comment:
Is there any reason to not use intersphinx on this link?

Reply:
https://github.com/readthedocs/readthedocs.org/pull/8283#discussion_r662533886

Reply:
Done!